src/utils/datoUtils.ts

```diff
@@ -309,3 +309,8 @@ export const isExpiredForhandsvarsel = (
 
   return false;
 };
+
+export function isDateBefore(date?: Date, targetDate?: Date): boolean {
```

Comment:
Litt rart å ha en sånn funksjon med to valgfri parametere. Kunne man gjort begge påkrevd?

Reply:
Godt poeng! Vi har fjerna funksjonen i datoUtils og heller lagt sjekken i en funksjon over komponenten.

Reply:
Det ble bedre :)